lib/style.css

```diff
@@ -56,6 +56,20 @@ body {
   background-color: <<BACK_PRIMARY>>;
 }
 
+#player.material paper-icon-button.playing::after {
+  content: '';
+  display: block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin-top: -1.2em;
+  height: 2.4em;
+  width: 2.4em;
```

Comment:
Ah, I see. We are using `z-index: -1` to act as a background. Any reason we don't do a `before`? We shouldn't need the `margin/z-index` adjustments then (`position: absolute` breaks us out of static flow).